pom.xml

```diff
@@ -43,6 +43,11 @@
       <artifactId>mockito-core</artifactId>
       <version>4.6.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
```

Comment:
I got rid of the `mockito` error messages by adding an extra dependency as mentioned in the GitHub discussion [here](https://github.com/mockito/mockito/issues/1325).